pgjdbc/src/test/java/org/postgresql/test/jdbc3/CompositeQueryParseTest.java

```diff
@@ -214,7 +231,22 @@ private static String toString(List<NativeQuery> queries) {
         sb.append(";/*cut*/\n");
       }
       sb.append(query.nativeSql);
+      System.out.println("TO STRING :" + query.nativeSql.toString());
     }
     return sb.toString();
   }
+
+  private static List<String> getOriginalSql(String query, boolean standardConformingStrings, boolean withParameters,
+      boolean splitStatements) {
+    try {
+      List<NativeQuery> queries = Parser.parseJdbcSql(query, standardConformingStrings, withParameters, splitStatements, false, true);
+      List<String> originalQueries = new ArrayList<>(queries.size());
+      for (NativeQuery nativeQuery : queries) {
+        originalQueries.add(nativeQuery.originalSql);
+      }
+      return originalQueries;
+    } catch (SQLException e) {
+      throw new IllegalStateException("Parser.parseJdbcSql: " + e.getMessage(), e);
```

Comment:
`+e.getMessage()` rarely helps. Typically you want adding the relevant method arguments and/or local variables to make it easier for the end-user to analyze the failure